conda-forge.yml

```
@@ -1,4 +1,5 @@
 build_platform:
   osx_arm64: osx_64
+  linux_aarch64: linux_64
```

Please also add `linux_ppc64le` while you're at it to keep both in sync.
```suggestion
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```